app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,47 @@
+<div class="row p-3">
+<div class="col-md-3"> </div>
+<div class= 'border border-1 mt-3 col-md-6 p-3 d-block mx-auto'>
+<h2 class="text-secondary mt-3 mb-3 text-center"> Sign up </h2>
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field mb-3 row">
+    <%= f.label :name, class: "col-sm-2 col-form-label" %><br />
+    <div class="col-sm-10">
+    <%= f.text_field :name, class: "form-control" %>
+    </div>
+  </div>
+
+  <div class="field mb-3 row">
+    <%= f.label :email, class: "col-sm-2 col-form-label" %><br />
+    <div class="col-sm-10">
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", class: "form-control" %>
+    </div>
+  </div>
+
+  <div class="field mb-3 row">
+    <%= f.label :password, class: "col-sm-2 col-form-label" %>
+    <% if @minimum_password_length %>
+    
+    <% end %>
+    <div class="col-sm-10">
+    <%= f.password_field :password, autocomplete: "new-password",class: "form-control" %>
+    </div>
+  </div>
+
+  <div class="field mb-3 row">
+    <%= f.label :password_confirmation, class: "col-sm-2 col-form-label" %><br />
+    <div class="col-sm-10">
+    <%= f.password_field :password_confirmation, autocomplete: "new-password", class: "form-control" %>
+    </div>
+  </div>
+
+  <div class="actions">
```

Comment:
- [ ] As the previous reviewer mentioned, please make sure these buttons are moved to the centre.
![image](https://github.com/francksefu/my_expended/assets/45460192/f23606ac-dcd1-48cc-9292-a292239a0a8e)